src/meal.js

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable radix */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable array-callback-return */
+
+async function listComments(itemId) {
+  const response = await fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/necgfXW8nUf9YW13OlZY/comments?item_id=${itemId}`,
+  );
+
+  const comments = await response.json();
+  const commentsList = document.getElementById('comments');
+  commentsList.innerHTML = '';
+  const title = document.createElement('h5');
+  title.style = 'text-align: center; padding: 20px 0';
+  const ul = document.createElement('div');
+  title.innerHTML = `<h5>Comments</h5><h5 id="counter">(${comments.length})</h5>`;
+  comments.map((comment) => {
+    ul.innerHTML += `<p style="width: 100%; display: flex; justify-content: space-between"><span><strong>${comment.username}:</strong> ${comment.comment}</span> <span>${comment.creation_date}</span</p> <br>`;
+  });
+
+  commentsList.append(title);
+  commentsList.append(ul);
+}
+
+async function addComment(itemId) {
```

Comment:
- [ ] It looks like here we could use ES6 best practices (in this case arrow function). 🐱